src/components/pickers/Suggestions/Suggestions.Props.ts

```diff
@@ -47,6 +47,14 @@ export interface ISuggestionsProps<T> extends React.Props<any> {
    * Dictates whether or not the searchForMore button is displayed.
    */
   moreSuggestionsAvailable?: boolean;
+  /**
+   * Used to indicate whether or not the suggestions are loading.
+   */
+  isLoading?: boolean;
+  /**
+   * The text to display while the results are loading.
+   */
+  loadingText?: string;
```

Comment:
it is weird that you have 2 places now with this defined. IBaseSuggestionsProps and ISuggestionsProps? Why doesn't ISuggestionsProps inherit so you only define it in one place?